docs/conf/australia/2024/tales.rst

```diff
@@ -0,0 +1,45 @@
+:template: {{year}}/generic.html
+
+Tales from the Trenches
+=======================
+
+**Where real stories meet tech docs.** This year, Write the Docs Australia is excited to introduce "Tales from the Trenches"—a new initiative inviting attendees to share their unique experiences and offer fresh perspectives. 
+
+We’re offering the usual `lightning talks`_ session on Day 1, but with "Tales from the Trenches" on Day 2, we’re setting three broad topics to help spark ideas and focus your stories. 
+
+Why join in?
+------------
+                                                                                                                                                                                                                                                                            
+Great things happen when we share what we know. Our community is made up of technical writers, programmers, support teams, developer advocates, content strategists, and more; and we all work in such different spaces—public, private, and open source. 
+
+Sometimes the best conversations are the ones that happen in the hallway between sessions. With "Tales from the Trenches," we’re bringing those gems onto the stage for everyone to hear and enjoy.
+
+Who’s it for?
+-------------
+                                                                                                                                                                                                                                                                            
+"Tales from the Trenches" is for everyone\! Whether you’re a natural on stage or this is your first time holding the mic, all are welcome. Some of us have great stories but aren’t sure what’s worth sharing, while others want to speak but don’t know where to start. 
+
+This is a relaxed, open space for all of us to share a story or two. We’re all peers here, learning from each other as we go.
+
+How It Works
+------------
+                                                                                                                                                                                                                                                                            
+- **Talks on Day 2**: "Tales from the Trenches" will close out Day 2 with 5-7 minute lightning-style talks—no slides, just you and your story.  
```

Comment:
Maybe update the schedule entry? And add a note in the lightning talk page?

Reply:
YES! I meant to do this and forgot.